src/trials/userId.js

```diff
@@ -1,7 +1,10 @@
 import { lang, MTURK } from '../config/main'
-import { getUserId, getTurkUniqueId } from '../lib/utils'
+import { getUserId, getStudyId, getTurkUniqueId } from '../lib/utils'
 import { baseStimulus } from '../lib/markup/stimuli'
 
+
+
+
 const userId = () => {
   if (MTURK) {
     return {
```

Comment:
is this file still needed?

Reply:
The file should not be needed anymore but it still contains some MTURK logic, so I'll delete it once me and Galen get the logic down.